<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketListener</class>
 <widget class="QWidget" name="SocketListener">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Socket Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblConnection">
          <property name="text">
           <string>Socket to listen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editSocket"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDisconnect">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkText">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbCodec"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblReplyType">
          <property name="text">
           <string>Reply Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbReplyType">
          <item>
           <property name="text">
            <string>No Reply</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Echo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Binary</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editReply"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
